apps/src/sites/studio/pages/levelbuilder_applab.js

```diff
@@ -25,6 +25,8 @@ $(document).ready(function () {
     "buzzer.off": null,
     "buzzer.stop": null,
     "buzzer.play": null,
```

Comment:
Should probably remove `buzzer.play` from this, the list of maker blocks for levelbuilder.